test/test_run_db_test_docker_db_check_arguments.py

```diff
@@ -45,8 +57,8 @@ def assert_mem_disk_size(self, mem_size: str, disk_size: str):
             return_code = exit_result[0]
         return_code = exit_result[0]
         self.assertEqual(return_code, 0)
-        self.assertIn(f"MemSize = {mem_size}", output)
```

Comment:
Does not work anymore since docker-db 7.1.7.
The result in /exa/etc/ExaConf is
...
MemSize = 1.2998 GiB
...
now.
Seems somebody converts the value to a float and back.

Reply:
In ITDE we use integer values (https://github.com/exasol/integration-test-docker-environment/blob/main/exasol_integration_test_docker_environment/test/test_test_environment_db_mem_size.py).
If you put None-Integer values in this test, it fails, too (I checked ;) ).

Reply:
I created a ticket in ITDE:
https://github.com/exasol/integration-test-docker-environment/issues/212

Reply:
I told Benedikt. He said it's a feature not a bug, as those values are converted to float internally anyway and now the real values are shown in EXAConf :)